.github/workflows/publish.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
+
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - run: npm ci
+      - run: npm test
+
+  publish-gpr:
+    needs: build
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          registry-url: https://npm.pkg.github.com/
+          scope: '@tl-its-umich-edu'
+      - run: npm ci
+      - run: npm publish
```

Comment:
Similarly.. 
```
  - run: |
       npm ci && npm publish 
```

Reply:
I'm not sure why most of the examples I've seen have separate steps for the npm commands, but UMGPT tells me that it could be useful for logging separate outputs, and that running both may be hard to diagnose which command failed.

Reply:
ok, You can keep it as is if it help in debugging purposes.